src/lease.ts

```diff
@@ -48,6 +48,32 @@ const enum State {
 
 /**
  * Lease is a high-level manager for etcd leases.
+ * Leases are great for things like service discovery:
```

Comment:
Typedoc in general is disappointing, but there's no way I'm going to manually typedef all those RPC interfaces.